rust-toolchain

```diff
@@ -1 +1 @@
-nightly
+nightly-2022-04-18
```

Comment:
Can we use a newer version? Say nightly-2023-01-*

Reply:
I think we can, but would rather do so in a follow-up PR.